lib/keycloak-api-rails/authentication.rb

```diff
@@ -28,7 +28,7 @@ def keycloak_authenticate
 
     def authentication_failed(message)
       logger.info(message)
-      [401, {"Content-Type" => "application/json"}, [ { error: message }.to_json]]
+      render status: :unauthorized, json: { error: "Invalid Token" }
```

Comment:
I am not sure to understand this change. 
The previous behavior changed: the error message is not exposed in the json payload anymore.